client/app/scripts/components/time-travel-timeline.js

```diff
@@ -284,12 +284,14 @@ class TimeTravelTimeline extends React.Component {
     // Ticks are disabled if they are in the future or if they are too transparent.
     const disabled = timestamp.isAfter(this.state.timestampNow) || opacity < 0.4;
     const handleClick = () => this.jumpTo(timestamp);
+    // Fixes Chrome effective line-height miscalculation in certain environments
+    const foreignObjectStyle = { lineHeight: '150%' };
```

Comment:
I think a bit more robust solution would be to set line height equal to the height of the foreign object, i.e. `{ lineHeight: '20px' }`. That would be a bit more self-explanatory and it would still work well if the font size ever changed.